backend/user_page/models/SearchHistory.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+from backend.user_page.models.User import User
+
+
+class SearchHistory(models.Models):
+    id = models.AutoField(primary_key=True)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
The user id must have the possibility of being null and I do not think we need the cascade delete here.